core/src/test/scala/restui/models/ContentTypeSpec.scala

```diff
@@ -0,0 +1,20 @@
+package restui.models
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.prop.TableDrivenPropertyChecks
+
+class ContentTypeSpec extends AnyFlatSpec with Matchers with TableDrivenPropertyChecks {
+  private val properties = Table(
+    ("file", "type"),
+    ("test.yaml", ContentType.Yaml),
+    ("test.yml", ContentType.Yaml),
+    ("test.json", ContentType.Json),
+    ("test.txt", ContentType.Plain)
```

Comment:
What if users try to pass a .conf file or something else ? Parser will eventually fail. I like to thing always that the users are dumb and they are always going to make mistakes. So I would say we should throw an Exception if requirments are not met and exit with a message that will be usefull for users to identify their problem.

Reply:
There is no parser involved. The contentType is purely for the HTTP response

Reply:
I'll remove it as it's unclear